torch/csrc/jit/runtime/profiling_graph_executor_impl.h

```diff
@@ -17,10 +22,12 @@ struct ProfilingGraphExecutorImpl : public GraphExecutorImplBase {
  private:
   void runProfilingInsensitiveOptimizations(std::shared_ptr<Graph>& graph);
   void runProfilingOptimizations(std::shared_ptr<Graph>& graph);
+  void replaceFallbackGraphWithFallbackFunction(Block* b);
   std::unique_ptr<ProfilingRecord> pr_;
   c10::optional<ExecutionPlan>
       profiling_plan_; // plan to run in order to profiling the code
   c10::optional<ExecutionPlan> optimized_plan_;
+  std::vector<std::unique_ptr<Function>> fallback_functions_;
```

Comment:
nit: maybe add a comment that these functions are owned by this interpreter, although i guess the unique_ptr makes that evident

Reply:
will add a comment